tests/vars/qubes-rpc.yml

```diff
@@ -1,6 +1,8 @@
 - policy: qubes.ClipboardPaste
   starts_with: |-
     ### BEGIN securedrop-workstation ###
+    @tag:sd-send-app-clipboard sd-app ask
```

Comment:
I also think using `sd-app` here makes sense: the intent of the `sd-client` tag is to support dev environment for opening in dispVM and using split GPG (see https://github.com/freedomofpress/securedrop-workstation/commit/784076da56d25f2c089864ae9cff2aafb0a756a0)

Reply:
We can revisit this later, should this be an issue in developer's environment. It should be a small diff that is easily applied on new and existing workstations